mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv

```diff
@@ -0,0 +1,4103 @@
+SourceListName,SourceFlowName,SourceFlowUUID,SourceFlowContext,SourceUnit,MatchCondition,ConversionFactor,TargetFlowName,TargetFlowUUID,TargetFlowContext,TargetUnit,Mapper,Verifier,LastUpdated
+ecoinvent_v3.6,"1,4-Butanediol",38a622c6-f086-4763-a952-7c6b3b1c42ba,air/urban air close to ground,kg,=,1,"1,4-Butanediol",34d53001-84a2-3053-b887-32b891e334eb,emission/air/troposphere/urban/ground-level,kg,Edelen,Hottle,8/21/2020
```

Comment:
@carl-vadenbo Please see the name used here. In the mapping files the SourceListName needs to match exactly the list name in the FlowList files, and the filenames of these files.  Please inform us of the standard name that you wish to use for this Ecoinvent flow list v3.6. thanks

Reply:
@WesIngwersen Thanks for spotting this inconsistency. I haven't been able to locate where "ecoinvent_v3.6" is present, but let's go for "ecoinventEFv3.6" in any case. Please just let me know, should you need any further actions from my side to fix this, thanks!

Reply:
@carl-vadenbo what about just `ecoinventv3.6` ? i think that would be more consistent with the other list names like `IDEAv2.2` . the `EF` is implicit

Reply:
@WesIngwersen sure, I'm fine with that, thanks!